indico/modules/events/contributions/util.py

```diff
@@ -107,7 +109,12 @@ def generate_spreadsheet_from_contributions(contributions):
                         'Session': c.session.title if c.session else None,
                         'Track': c.track.title if c.track else None,
                         'Materials': None,
-                        'Presenters': ', '.join(speaker.full_name for speaker in c.speakers)}
+                        'Presenters': ', '.join(speaker.full_name for speaker in c.speakers),
+                        'Authors': ', '.join(author.full_name for author in c.submitters
+                                             if author.author_type == AuthorType.primary),
+                        'Co-Authors': ', '.join(author.full_name for author in c.submitters
+                                                if author.author_type != AuthorType.primary),
+                        'Boardnumber': c.board_number}
```

Comment:
I think spaces in CSV column headers are OK, so I'd rather go for `'Board number'` here.

"Boardnumber" sounds a bit "denglisch" ;)